### ISSUE TITLE
Skiff Android is extremely slow and almost unusable due to large inbox

### ISSUE THREAD

**Issue by riverar** [skiff-org/skiff-apps#117]
**Describe the bug**
I have ~60K emails in my Inbox, from an Outlook import where I was, for better or worse, able to treat the Inbox as a Twitter timeline stream.

**To Reproduce**
1. Have over 60K messages in your Inbox
2. Attempt to us the Skiff Android application
3. Observe opening the hamburger menu, any email, or generally interacting with the app takes ~1 min each time

**Expected behavior**
Fast user experience 🪄

**Desktop**
Unaffected

**Smartphone**
 - Device: Pixel 7
 - OS: Android 13
 - Browser: N/A
 - Version: N/A

**Additional context**
Skiff Mail (non-beta) 99.0

**Comment by amilich** [comment]
Looking into this

**Comment by ahmadassayyaaf** [comment]
Its slow to my smartphone to, my device is Redmi 9C

**Comment by Pharoh-Ramses** [comment]
Agreed, it is almost unusable. I experience slow behavior on the desktop and web client as well. Search takes forever and does not always work. I have to go back into the gmail client for the same search. Google finds the email immediately. Mind you, these searches are not for old email, maybe 2 weeks old tops. Also keyboard shortcuts don't register.

**Comment by jgdevelopment** [comment]
Yeah that definitely should not be happening. We'll continue to investigate the root cause. Even with 10,000s of emails loading and searching should be fast.

**Comment by ericboutet** [comment]
I do not know if this can help, but I got the same behavior when moving from gmail. I had over 175 labels imported and moved them (by hand) to folder. Once I cleaned up, the hamburger menu becomes responsive. It seems related to the quantity of label and not the quantity of email in my case.

**Comment by riverar** [comment]
At this point, it should be easy for Skiff to reproduce this. If they can't, they should communicate that to us so we can send logs or other debug information.

As it stands right now, the import story is just broken/bad.

Example: I get parcel emails (with barcodes) and I can't open the lockers until I fire up the app, wait forever, search for the package email, wait forever, tap the email, wait forever, then scan the code.
